modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/display/context/DLViewEntriesDisplayContext.java

```diff
@@ -366,6 +369,20 @@ public boolean isVersioningStrategyOverridable() {
 		return _dlAdminDisplayContext.isVersioningStrategyOverridable();
 	}
 
+	private String _addDoAsUserIdParameter(
+		ThemeDisplay themeDisplay, String url) {
+
+		if ((themeDisplay != null) &&
```

Comment:
I believe themeDisplay cannot be null here, so the condition may be simplified a little bit.

Reply:
Changed, in both places